hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -110,9 +133,19 @@ def commit(self, sim_stalled: bool) -> None:
         # If error bits are set, pending_halt should have been set as well.
         assert self._err_bits == 0
 
-        if self._new_rnd_data:
-            self.wsrs.RND.set_unsigned(self._new_rnd_data)
-            self._new_rnd_data = None
+        # RTL must send a RND done signal 5 cycles after calculation
+        assert self.rnd_cdc_counter
+
+        # Stall until getting the RND done signal from RTL, then set RND register
```

Comment:
Is this right when we're waiting on a `RND_PREFETCH`? In that case, I would have thought we could execute instructions despite the fact that there's RND data coming in?